dolphinscheduler-registry/dolphinscheduler-registry-plugins/dolphinscheduler-registry-etcd/src/test/java/org/apache/dolphinscheduler/plugin/registry/etcd/EtcdRegistryTest.java

```diff
@@ -140,4 +141,23 @@ public void notify(Event event) {
     public static void after() throws IOException {
         registry.close();
     }
+
+    @Disabled
+    @Test
```

Comment:
What's the point to add a `@Disabled` test?

Reply:
This test method is used for the user to test connect etcd with configuration, use need to input their endpoints and SSL file.

Reply:
> This test method is used for the user to test connect etcd with configuration, user need to input their endpoints and SSL file.

Etcd has its own method to test connection, users don't even have the source codes, this test should be removed, and the comments should go into the doc